examples/meteor/client/styles/base.css

```diff
@@ -285,9 +284,9 @@ label[for='toggle-all'] {
 	display: block;
 }
 
-#todo-list li .edit {
-	display: none;
-}
+/*#todo-list li .edit {*/
+	/*display: none;*/
+/*}*/
```

Comment:
I'm not familiar with how you guys update the codebases. If you bump todomvc-common, will it re-create bower folders, etc? Meteor's folder structure doesn't work that way. How I've restructured this is really the way it ought to be structured for Meteor.